Importer/jctdata/datasources/crossref.py

```diff
@@ -60,6 +60,11 @@ def gather(self):
                     title = entry.get("title")
                     issns = entry.get("ISSN")
                     issns = list(set(issns))    # because sometimes the issns are duplicated
+                    if "0000-0000" in issns:
```

Comment:
I know we've only seen this one in the wild so I'm happy to merge this, but it might be worth generalising to a list of ISSNs to filter, e.g. for variations 00000000 and XXXX-XXXX